examples/drivers/windows/otLwf/command.c

```diff
@@ -127,6 +134,12 @@ otLwfCmdInitialize(
             break;
         }
 
+        NtStatus = otLwfCmdResetDevice(pFilter, FALSE);
+        if (!NT_SUCCESS(NtStatus))
+        {
+            Status = NDIS_STATUS_FAILURE;
+        }
+
     } while (FALSE);
```

Comment:
The do {...} while (FALSE) loop is just a way to easily provide a method to exit out on failures (via break).